### PR TITLE
[Modular] Gives shadekin A BUNCH of stuff to make them closer to being accurate to their origination lore.

### PR DESCRIPTION
**Pull request by Odairu**
## About The Pull Request
Gives shadekin nightmare regen at half effectiveness, blocks stamina regen in the light, gives a movespeed debuff in the light, gives innate nobreath, and nightvision, makes them weak to flashes

## Why It's Good For The Game
further makes shadekin unique by pushing them towards being more lore accurate for their origins

## Proof Of Testing
image
I waited for several minutes and didn't regen until I made it into the dark, source: dude trust me
Screenshot_13

## Changelog

:cl:
add: Darkness regen, nightvision, nobreath, flash vulnerability, movespeed decrease in light, no stamina regen in light
/:cl:

### CODE REVIEW

File: modular_zubbers/code/modules/mob/living/carbon/human/species_types/shadekin.dm
Comment (Majkl-J) [inline modular_zubbers/code/modules/mob/living/carbon/human/species_types/shadekin.dm:1]: We really need to standardize where we put our species code

**Review by Maia-J — COMMENTED** [review]
Gonna TM this and see how the people feel about it

**Comment by Odairu** [comment]
> seconding the slow being agonizing, and totally stunlocked in light is straight up busted (as someone who plays a secmain shadekin this PR makes it impossible to engage with antags in almost every situation) I'm all for playing a balanced race that's gimped in some way but this feels unplayable as-is

Ill probably remove the stamina thing because I cant figure out how to just delay the stamina regen in favor of like, blindness when over a certain light level, ill see about boosting speed in darkness

**Comment by Odairu** [comment]
> > Ill probably remove the stamina thing because I cant figure out how to just delay the stamina regen in favor of like, blindness when over a certain light level, ill see about boosting speed in darkness
> 
> Oooh boy.. I'll let you cook on the blindness. That SOUNDS iffy.

Tgere needs to be something balancing out giving them no breath and noct regen, people were already mad over global comms like abductors, so w.o downsides the 2 need to be scrapped tbqh

**Comment by SidelSin** [comment]
> > > Ill probably remove the stamina thing because I cant figure out how to just delay the stamina regen in favor of like, blindness when over a certain light level, ill see about boosting speed in darkness
> > 
> > 
> > Oooh boy.. I'll let you cook on the blindness. That SOUNDS iffy.
> 
> Tgere needs to be something balancing out giving them no breath and noct regen, people were already mad over global comms like abductors, so w.o downsides the 2 need to be scrapped tbqh

The slowness overall feels really jarring, and complete darkness is very hard to come by on-station, barring maintenance. It takes a conscious decision to actually capitalize on nocturnal regeneration too. Even with the station in red-alert, and only emergency lights up, you can barely passively take advantage of it because there's always at least one source of glow near you. So you have turn off your light, and function in low visibility. Or in the case of bright, high traffic areas, just go off into maintenance and wait. And even then, there are some maps with lighting in maintenance. 

Speed is key to a lot of jobs (paramedic, shaft miner, security, even engineering when disaster strikes), and so you're always naturally more gimped than any other crewmember. Compared to pod-people who pretty much benefit from healing all the time for basically free, golems who have slowness but innate armor, and even snails which are slower than anything mentioned but have the shell, being _mildly_ better in the dark really feels tacked on when you can just go stamcrit in the hall and never recover. 

And imo, another nobreath species feels cheap when it comes to uniqueness. I'd rather the buffs just be scrapped, and if you really want to have something as a downside for the empathy and all? Just have flash weakness and _maybe_ mood debuffs/buffs in light/dark to manage. Slowness just sucks to deal with.

**Comment by DancingAlpacas** [comment]
This feels really bad to play as a frequent shadekin player. I don't like the fact that I'm unable to really act properly to things such as trying to help someone being attacked because of my light speed debuff, and the fact that because of the light bug that any little shove or knockdown permanently makes me unable to get up is really, really bad. I get and understand that your goal is to achieve diversity with this race but these changes are really counter-intuitive for the game, there's a reason stuff like nightmare is only antag and not a playable race. Podpeople get to heal for free and do not need to eat, moths get a free jumpboot dash and climbing rope, felinids can land on their feet/lick wounds, but all shadekin get is essentially nerfed slimelink. I wouldn't really be afraid to give them more bonuses as they already have frail, even teshari were tweaked to be able to walk on tables now to offset their fragility. Also a bit of an OOC note here but there's multiple versions of shadekin lore that I personally have seen quite a few shadekin players run with their own version of whichever they choose and these being heavily based on the Cit one I believe feels limiting. Please reconsider this light debuff and the changes.

File: modular_zubbers/code/modules/mob/living/carbon/human/species_types/shadekin.dm
Comment (Odairu) [reply modular_zubbers/code/modules/mob/living/carbon/human/species_types/shadekin.dm:1]: where do you want me to move it ;w;

**Comment by Odairu** [comment]
> This feels really bad to play as a frequent shadekin player. I don't like the fact that I'm unable to really act properly to things such as trying to help someone being attacked because of my light speed debuff, and the fact that because of the light bug that any little shove or knockdown permanently makes me unable to get up is really, really bad. I get and understand that your goal is to achieve diversity with this race but these changes are really counter-intuitive for the game, there's a reason stuff like nightmare is only antag and not a playable race. Podpeople get to heal for free and do not need to eat, moths get a free jumpboot dash and climbing rope, felinids can land on their feet/lick wounds, but all shadekin get is essentially nerfed slimelink. I wouldn't really be afraid to give them more bonuses as they already have frail, even teshari were tweaked to be able to walk on tables now to offset their fragility. Also a bit of an OOC note here but there's multiple versions of shadekin lore that I personally have seen quite a few shadekin players run with their own version of whichever they choose and these being heavily based on the Cit one I believe feels limiting. Please reconsider this light debuff and the changes.

I did remove the stamina portion of it, whenever that gets put through if it hasn't already

**Review by Maia-J — COMMENTED** [review]
I forgot this existed, how do you feel about its current state?

**Comment by ShockOmega** [comment]
I'm just going to add my own two cents into this. I play shadekin a lot. I mean *A LOT*. Its my most played species in fact.
While I really love that they don't have to breathe now and have flash sensitivity, the slow down effect is incredibly noticeable and very jarring. This is doubly so with any job that relies on a MOD or speed to get a lot of stuff done. This includes security, paramed, and engineering to name a few. I tested out how it felts to use an atmos mod suit and it felt like I was a total snail.

If we are going for lore accurate (looking at Cit): empathy, brute and burn weakness, breathless, marish, and light sensitivity makes sense. The whole...slow down in the light doesn't. It makes shadekin irritating to play. 

The whole darkness power stuff also isn't enough to offset the utter pain it is to play with such a slow down. Its so incredibly situational and the only ones who could benefit from it on the regular would be miners; however, shadekin's are weak to brute and burn so I don't see this as an issue.

Breathless is genuinely not a huge buff. Again the only people who truly benefit from it on a regular are miners. But seeing as oxy tanks last for so long you rarely need to change, its not that big of a buff and fits straight in line with shadekin lore.

**Comment by Odairu** [comment]
> I'm just going to add my own two cents into this. I play shadekin a lot. I mean _A LOT_. Its my most played species in fact. While I really love that they don't have to breathe now and have flash sensitivity, the slow down effect is incredibly noticeable and very jarring. This is doubly so with any job that relies on a MOD or speed to get a lot of stuff done. This includes security, paramed, and engineering to name a few. I tested out how it felts to use an atmos mod suit and it felt like I was a total snail.
> 
> If we are going for lore accurate (looking at Cit): empathy, brute and burn weakness, breathless, marish, and light sensitivity makes sense. The whole...slow down in the light doesn't. It makes shadekin irritating to play.
> 
> The whole darkness power stuff also isn't enough to offset the utter pain it is to play with such a slow down. Its so incredibly situational and the only ones who could benefit from it on the regular would be miners; however, shadekin's are weak to brute and burn so I don't see this as an issue.
> 
> Breathless is genuinely not a huge buff. Again the only people who truly benefit from it on a regular are miners. But seeing as oxy tanks last for so long you rarely need to change, its not that big of a buff and fits straight in line with shadekin lore.

the slowdown is gonna be changed to like 10% rather than the 50% its at now

**Review by Maia-J — APPROVED** [review]

**Pull request merged**